includes/amp-helper-functions.php

```diff
@@ -429,12 +442,50 @@ function amp_prepare_comment_post() {
 			if ( is_wp_error( $error ) ) {
 				$error = $error->get_error_message();
 			}
-			$error = strip_tags( $error, 'strong' );
-			wp_send_json( compact( 'error' ) );
+			$amp_mustache_allowed_html_tags = array( 'strong', 'b', 'em', 'i', 'u', 's', 'small', 'mark', 'del', 'ins', 'sup', 'sub' );
```

Comment:
Oh nice one!

Reply:
@westonruter this is a little hard to test as it's on a die handler and causes much issues. I left it out of the burrent test. Perhaps you could give a suggestion?